frontend/packages/kubevirt-plugin/src/components/LabelsList/labels-list.tsx

```diff
@@ -0,0 +1,85 @@
+import * as React from 'react';
+import {
+  Grid,
+  GridItem,
+  Text,
+  TextVariants,
+  Button,
+  Split,
+  SplitItem,
+} from '@patternfly/react-core';
+import { PlusCircleIcon } from '@patternfly/react-icons';
+import { ExternalLink, resourcePath } from '@console/internal/components/utils';
+import { K8sResourceKindReference } from '@console/internal/module/k8s';
+import { LabelRow } from './LabelRow/label-row';
+import { ADD_LABEL, LABEL_KEY, LABEL_VALUE } from './consts';
+import { IDLabel } from './types';
+import './labels-list.scss';
+
+export const LabelsList = <T extends IDLabel>({
```

Comment:
you can default the type to IDLabel here

Reply:
how would you do that?